components/is-analytics/org.wso2.analytics.solutions.is.analytics/widgets/IsAnalyticsAverageSessionDuration/src/resources/widgetConf.json

```diff
@@ -10,19 +10,16 @@
     },
     "providerConfig": {
       "configs": {
-        "type": "RDBMSBatchDataProvider",
+        "type": "SiddhiStoreDataProvider",
         "config": {
-          "datasourceName": "IS_ANALYTICS_DB",
+          "siddhiApp": "@App:name(\"IS_ANALYTICS_SESSION_DATA\") @App:description(\"Receiving Streams from IS and executing and analyzing statistics for session\") define stream HandleSessionInformation (meta_tenantId int, sessionId string, startTimestamp long, renewTimestamp long, terminationTimestamp long, endTimestamp long, duration long, isActive bool, username string, userstoreDomain string, remoteIp string, region string, tenantDomain string, serviceProvider string, identityProviders string, rememberMeFlag bool, userAgent string, usernameWithTenantDomainAndUserstoreDomain string, timestamp long); @store(type='rdbms', datasource='IS_ANALYTICS_DB') @primaryKey('meta_tenantId', 'sessionId') define table SessionInformationTable ( meta_tenantId int, sessionId string, startTime string, terminateTime string, endTime string, duration long, isActive bool, username string, userstoreDomain string, remoteIp string, region string, tenantDomain string, serviceProvider string, identityProviders string, rememberMeFlag bool, userAgent string, usernameWithTenantDomainAndUserstoreDomain string, currentTime string, startTimestamp long, renewTimestamp long, terminationTimestamp long, endTimestamp long, timestamp long);",
```

Comment:
Please add only the store definition in siddhi app. That will include only @store, @primarykey, @Index and table definition. Remove all other processing logic